recipe/meta.yaml

```diff
@@ -1,42 +1,50 @@
 {% set name = "fast-pt" %}
-{% set version = "3.1.1" %}
+{% set version = "3.2" %}
+{% set python_min = "3.5" %}
```

Comment:
```suggestion
```

I doubt you actually want conda to be able to install this package with python 3.5 in conda.